pkg/operator/ceph/config/datapath.go

```diff
@@ -60,12 +63,12 @@ func NewStatefulDaemonDataPathMap(
 	daemonType DaemonType, daemonID, namespace string,
 ) *DataPathMap {
 	return &DataPathMap{
-		HostDataDir:       path.Join(dataDirHostPath, daemonDataDirHostRelativePath),
-		ContainerDataDir:  cephDataDir(daemonType, daemonID),
-		HostLogDir:        path.Join(dataDirHostPath, namespace, "log"),
-		ContainerLogDir:   VarLogCephDir,
-		HostCrashDir:      path.Join(dataDirHostPath, namespace, "crash"),
-		ContainerCrashDir: VarCrashCephDir,
+		HostDataDir:        path.Join(dataDirHostPath, daemonDataDirHostRelativePath),
+		ContainerDataDir:   cephDataDir(daemonType, daemonID),
+		HostLogAndCrashDir: path.Join(dataDirHostPath, namespace, "log"),
```

Comment:
I was thinking this would be `path.Join(dataDirHostPath, namespace)`, and `log` or `crash` would be added to this automatically by functions.

Reply:
It's possible that `HostLogAndCrashParentDir` would be a better name or something easier to understand for others.